piglow/examples/strobe.go

```diff
@@ -0,0 +1,47 @@
+package main
+
+import (
+	"time"
+
+	"github.com/goiot/devices/piglow"
+
+	"golang.org/x/exp/io/i2c"
+)
+
+func main() {
+	piglow, err := piglow.Open(
+		&i2c.Devfs{
+			Dev:  "/dev/i2c-1",
+			Addr: 0x54,
+		},
+	)
+	if err != nil {
+		panic(err)
+	}
+
+	defer piglow.Close()
```

Comment:
I'd call piglow p, not avoid conflict with the package. Those who don't have enough context about the variable piglow may and lazy enough to scroll up to read it may get confused.